Classes/WobbleView.swift

```diff
@@ -9,23 +9,23 @@
 import UIKit
 import QuartzCore
 
-public class WobbleView: UIView {
+open class WobbleView: UIView {
     
     /*
     The frequency of oscillation for the wobble behavior.
     */
-    @IBInspectable public var frequency: CGFloat = 3
+    @IBInspectable open var frequency: CGFloat = 3
     
     /*
     The amount of damping to apply to the wobble behavior.
     */
-    @IBInspectable public var damping: CGFloat = 0.3
+    @IBInspectable open var damping: CGFloat = 0.3
     
     /*
     A bitmask value that identifies the edges that you want to wobble.
     You can use this parameter to wobble only a subset of the edges of the rectangle.
     */
-    @IBInspectable public var edges: ViewEdge = .Right
+    @IBInspectable open var edges: ViewEdge = .Right
```

Comment:
It doesn't work with Swift 4.0. The error message is "_Property cannot be marked @IBInspectable because its type cannot be represented in Objective-C_".